src/index.css

```diff
@@ -879,12 +879,13 @@ form label {
   display: -webkit-box;
   display: -ms-flexbox;
   display: flex;
-  width: 22%;
+  align-items: center;
+  width: 30%;
 }
 
 .dashboard-top-approve-reject button {
   margin: auto 2%;
-  width: 45%;
+  /* width: 45%; */
```

Comment:
Could you remove this line instead of commenting it out?

Reply:
Sure. I tried to touch the .css as little as possible.